indexer/httpd/src/graphql/subscription/event.rs

```diff
@@ -39,11 +51,20 @@ impl EventSubscription {
             .map(|block| block.block_height)
             .unwrap_or_default();
 
+        let block_range = match since_block_height {
+            Some(block_height) => block_height as i64..=latest_block_height,
+            None => latest_block_height..=latest_block_height,
+        };
+
+        if block_range.try_len().unwrap_or(0) > MAX_PAST_BLOCKS {
+            return Err(async_graphql::Error::new("since_block_height is too old"));
```

Comment:
In error messages, tracing messages, or comments/docstrings, can you always use backticks around variable names:

```suggestion
            return Err(async_graphql::Error::new("`since_block_height` is too old"));
```